### ISSUE TITLE
NVDA Announcement Issue

### ISSUE THREAD

**Issue by RituAdhikari** [espeak-ng/espeak-ng#818]
Announcing on date format (dd/ mm/yyyy) mm when language selected as Great Britain but when language is united state it is announcing correctly
Announcing No  as number when language selected as United states but when language is great Britain it is announcing "no" correctly

**Comment by sukiletxe** [comment]
Cannot confirm, both in NVDA 2020.2 and 2020.3 beta 1. Can you give more details (NVDA version, synth, voice, complete examples where it fails?)
I've tested with strings "31/12/2020" and "No", and the results are correct in English Great Britain. I'm using the Male3 variant.

**Comment by valdisvi** [comment]
@RituAdhikari, from your description it is not exactly clear what the problem is. If I test example given by @sukiletxe  e.g. ["31/12/2020. No 1."](https://odo.lv/Espeak?eslang=-ven-gb&speed=160&output=-w&debug=-x&text=31%2F12%2F2020.+No+1.&formmethod=get) I cannot see anything wrong neither with GB non US variants of the language.
If you use screen reader, check that it doesn't do anything strange before it pass text to eSpeak NG.

**Comment by Qchristensen** [comment]
As noted, it would be good to get more information on exactly what is happening and what is expected here.

Regardless of whether I had eSpeak set to Engligh (American) or English (United Kingdom), with NVDA, I get:

"no" and "No" both read as the word "no"
"05/01/2020" reads as "zero-five slash zero-one, slash, two-thousand and twenty".

For the original poster - what is your Windows language set to?  That might be determining how NVDA reads dates.

**Comment by jaacoppi** [comment]
Closed until more information is received from the original author.
